### PR TITLE
Fix for alphanumeric filenames

### PR DESCRIPTION
**Pull request by Harsh-br0**
Hope it shouldn't raise pep8 (. ❛ ᴗ ❛.)

### CODE REVIEW

**Comment by Harsh-br0** [comment]
Finally it raised

File: userge/utils/tools.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def check_numerical_order(a: str, total: int) -> Union[float, str]:
     if r:
         return float(r)
     nval = ord(a[0])
-    return total + nval if total > nval else nval
+    return 10**8*total + nval
```
Comment (Krishna-Singhal) [inline userge/utils/tools.py:49]: it will add the first str file name to end

**Comment by Krishna-Singhal** [comment]
When Fixing Pep8 🤔🤔

**Comment by Harsh-br0** [comment]
> When Fixing Pep8 🤔🤔

When give mercy to those peeps who needs compressed audio files 🤔🤔

**Comment by rking32** [comment]
nice :+1:

**Comment by Krishna-Singhal** [comment]
still not a permanent fix 😧

**Pull request closed without merging**